blockbase/readme.md

```diff
@@ -0,0 +1,31 @@
+# === Blank Canvas Blocks ===
+Contributors: Automattic
+Requires at least: ? 
+Tested up to: ?
+Requires PHP: ? 
+License: GPLv2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+An unopinionated 'Ponyfill' to fill the gaps between where Gutenberg IS and where Gutenberg is GOING.
+
+## == Description ==
+
+The purpose of Blank Canvas Blocks is to provide a "ponyfill" that allows for 100% "configuration expression" of design.  Any aspect of configurable design that Gutenberg does not yet take into account is expressed in theme.json 'custom' properties.
```

Comment:
We'd have to search and replace `BCB` and `Blank Canvas Blocks` too

Reply:
Done